snorkel/classification/training/loggers/checkpointer.py

```diff
@@ -199,6 +198,22 @@ def load_best_model(self, model: SnorkelClassifier) -> SnorkelClassifier:
 
         return model
 
+    def _validate_config(self) -> None:
+        if self.config.checkpoint_dir is None:
```

Comment:
`None` isn't a valid input anymore, so probably don't need this. Shouldn't need to throw exceptions for every contract violation.